server/datastore/mysql/migrations/tables/20240129162819_AddPrefixToWindowsUpdatesProfiles_test.go

```diff
@@ -0,0 +1,65 @@
+package tables
+
+import (
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+	"github.com/stretchr/testify/require"
+)
+
+func TestUp_20240129162819(t *testing.T) {
+	db := applyUpToPrev(t)
+
+	prof1UUID := "w" + uuid.NewString()
+	prof1UpdatedAt := time.Now().UTC().AddDate(0, 0, -3).Truncate(time.Second)
+	updateProfUUID := uuid.NewString()
+	updateProfUpdatedAt := time.Now().UTC().AddDate(0, 0, -3).Truncate(time.Second)
+
+	profStmt := `
+		INSERT INTO mdm_windows_configuration_profiles VALUES
```

Comment:
I'd suggest listing the column names before `VALUES` as it makes reading this hard, you have to go check the order of columns in the DB.

Reply:
Ah fair enough 👍